### DISCUSSION TITLE
How to read and paging a time serie in reverse timestamp order?

### DISCUSSION THREAD

**Discussion by ynnova** [ravendb/ravendb#15256] [General]
i.e. the last entry on top.

**Comment by ayende** [comment]
We don't provide a way to reverse the order of the results.
You can page easily via something like:

`from Users select timeseries(from HeartRate last 1 min)`

**Reply by ayende** [reply]
For further details, we don't provide the reverse sort order since we store the data in ascending order.
To do ascending work we'll need to decompress the data, and then reverse it. You can do the same on your end as well.

**Reply by lorenzomelato** [reply]
Tks Oren. I understand. 
In my case I have solved with an index as in any case my use case was to search for particular values in the timeserie.
